zeebe/broker/src/main/java/io/camunda/zeebe/broker/exporter/stream/ExporterContainer.java

```diff
@@ -44,14 +45,18 @@ final class ExporterContainer implements Controller {
   private ExporterMetrics metrics;
   private ActorControl actor;
 
-  ExporterContainer(final ExporterDescriptor descriptor, final int partitionId) {
+  ExporterContainer(
+      final ExporterDescriptor descriptor, final int partitionId, final ExporterPhase phase) {
     context =
         new ExporterContext(
             Loggers.getExporterLogger(descriptor.getId()),
             descriptor.getConfiguration(),
             partitionId);
 
     exporter = descriptor.newInstance();
+    if (phase == ExporterPhase.SOFT_PAUSED) {
+      softPauseExporter();
+    }
```

Comment:
:wrench: No strong opinion. But I would  do this either in `initContainer` or explicitly call `softPauseExporter` from `ExporterDirector` where `initContainer` is called.